pyproject.toml

```diff
@@ -1,5 +1,4 @@
 [tool.pytest.ini_options]
-addopts = "-q"
```

Comment:
This will have to be removed from the cookiecutter as well: I don't think we want pytest options in multiple places (in `pyproject.toml`, in the `PYTEST_ADDOPTS` envvar, as `pytest` command line arguments in `tox.ini`, ...). Let's keep them all in one place in the `PYTEST_ADDOPTS` default value in `tox.ini`. Locally, a developer can export their own `PYTEST_ADDOPTS` envvar in their shell and it will override the default opts, providing a convenient way to pass custom command line opts to `pytest` when just running `make` or `tox` commands (without having to run the `pytest` command directly/manually).